slice/Ice/OperationMode.ice

```diff
@@ -30,8 +29,8 @@ module Ice
         ["swift:identifier:nonmutating"]
         Nonmutating,
 
-        /// An idempotent operation. Invoking such an operation twice in a row must result in the same object state
```

Comment:
It's really about retries and at-most-once semantics, not about the standard/math definition of idempotent.